### ISSUE TITLE
[Bug] 1 pixel high native titlebar visible even when turned off

### ISSUE THREAD

**Issue by plutonium-239** [SpikeHD/Dorion#268]
**Describe the bug**
Even when the setting for native titlebar is turned off, a 1-pixel high bar is visible which looks pretty ugly (screenshot attached)

**To Reproduce**
Update to v6.2.0 and turn off native titlebar

**System Info (please complete the following information):**
 - OS: Windows 10
 - Version: v6.2.0
 - Extra Client Mods/Plugins: None (confirmed with Vencord turned off)
 - Any other relevant info: CSS cache cleared

**Additional context/Screenshots**

![image](https://github.com/user-attachments/assets/59773ade-84fe-4c33-b83a-9187dbe3560e)

**Comment by SpikeHD** [comment]
Submitted upstream issue https://github.com/tauri-apps/tao/issues/997

**Comment by Banaanae** [comment]
This should be fixed now
(Patched in tao 0.30.6, we're on 0.30.8)

**Comment by SpikeHD** [comment]
*Should* be, but from my testing there was still issues where some sides (top, left) were proper, and some (right, bottom) were not.

If you have used an actions build and can confirm that these issues are solved, let me know, otherwise this issue will stay open

**Comment by plutonium-239** [comment]
This has grown to 2 sides (top, left) after updating to v6.3.0 😢

![image](https://github.com/user-attachments/assets/12d300c2-f5f8-48ac-85a5-993a5b97adf9)

**Comment by plutonium-239** [comment]
Hey @SpikeHD can you reproduce this issue?

Is this because of tauri too?

**Comment by SpikeHD** [comment]
Yes this is happening for me too, it is a Tauri/Wry/Tao problem. I am waiting for them to update so I can test before I submit another bug report, as they seem to have fixed some behaviour regarding windows with both decorations off and shadows on - which Dorion does.

**Comment by SpikeHD** [comment]
This "bug" is an optional feature, just turn off "Show accent color on title bars and window borders" in Windows settings and it goes away :P

**Comment by h-lunah** [comment]
lol so it's actually a Windows "feature" that does it

didn't know that

**Comment by plutonium-239** [comment]
wouldn't say it's windows' fault here though, this is a bug in tauri/ecosystem since every other program (that needs to) has been able to avoid having the default borders without issue for a good number of years

**Comment by mangkoran** [comment]
It seems I just encountered this issue after I upgrade to v6.4.0

![image](https://github.com/user-attachments/assets/bb4b9da8-a9bd-4c80-9e40-a7af1ce4aee1)

I didn't encounter this issue on previous version < v6.4.0

![image](https://github.com/user-attachments/assets/ea4f6215-b9a6-45e7-ab56-e0119cad6172)

> This "bug" is an optional feature, just turn off "Show accent color on title bars and window borders" in Windows settings and it goes away :P

I have turned off the setting since forever.

![image](https://github.com/user-attachments/assets/a2bac662-a180-4287-a564-e05eb4281d3a)

**Comment by SpikeHD** [comment]
Actually you (and most - if not all - other Windows users) are experiencing the issue in both versions, but in v6.3.0 it's showing as black, not white

**Comment by SpikeHD** [comment]
([this build](https://github.com/SpikeHD/Dorion/actions/runs/13553395957) should work)

**Comment by SpikeHD** [comment]
Tested and working in a VM, gonna assume this works now elsewhere

**Comment by mangkoran** [comment]
I can confirm this bug has been fixed with https://github.com/SpikeHD/Dorion/releases/tag/v6.5.0. Thank you for your hard word @SpikeHD :D

**Comment by SpikeHD** [comment]
Haha yes, that should be "bump" and not "dump"

**Comment by plutonium-239** [comment]
Thanks!

With v6.5.0, this is the current state (now borders are everywhere except the top 😅):

![Image](https://github.com/user-attachments/assets/666bacec-d9bf-4320-820b-9ca37c5b1d23)

Is there a dorion setting to make it like before (no borders anywhere)? (Not the windows setting)

**Comment by SpikeHD** [comment]
There is not a setting for it, it's either everywhere or nowhere (by using the Windows setting) :P

**Comment by plutonium-239** [comment]
So currently it is still bugged? Is it still because of `wry`/`tauri`?

**Comment by SpikeHD** [comment]
Yes, it's still bugged (for you, not for my VM or that other person for some reason haha), and yes, it'd be a `wry`/`tauri` problem

**Comment by mangkoran** [comment]
This is how it looks like for me.

![Image](https://github.com/user-attachments/assets/3049d4cb-45f2-45fc-bcea-a58316476208)

I think the borders actually still there, but now it's in black or dark color [similar](https://github.com/SpikeHD/Dorion/issues/268#issuecomment-2574321749) to v6.3.0? Either way I didn't notice it now, just like v6.3.0.

> but in v6.3.0 it's showing as black, not white

**Comment by SpikeHD** [comment]
Could be a difference between Win10 and Win11, the VM I tested on is Win11 ¯\\\_(ツ)_/¯
